packetbeat/docs/reference/configuration/runconfig.asciidoc

```diff
@@ -0,0 +1,24 @@
+[[configuration-run-options]]
+=== Run Options
+
+{beatname_uc} can drop privileges after creating the sniffing socket.
+Root access is required for opening the socket, but everything else requires no
+privileges. Therefore, it is recommended that you have {beatname_uc} switch users after
+the initialization phase. The `uid` and `gid` settings set the User Id and Group
+Id under which {beatname_uc} runs.
+
+WARNING: On Linux, Setuid doesn't change the uid of all threads, so the Go
+         garbage collector will continue to run as root. Also note that process
+         monitoring only works when running as root.
+
+Example configuration for the `runoptions` section of the +{beatname_lc}.yml+ config file:
+
+[source,yaml]
+------------------------------------------------------------------------------
+packetbeat:
```

Comment:
Under the 1.x release, run option are not under the packbeat namespace yet. Did we push this also the the 1.3 branch?